rollup.config.js

```diff
@@ -887,9 +887,12 @@ function copyToPlaygrounds() {
         await fse.copy(writtenDir, destDir);
 
         // tickle live reload by touching the server entry
-        let serverEntry = ["entry.server.tsx", "entry.server.jsx"].find(
-          (entryPath) =>
-            fse.existsSync(path.join(playgroundDir, "app", entryPath))
+        let serverEntry = [
+          "entry.server.tsx",
+          "entry.server.js",
+          "entry.server.jsx",
+        ].find((entryPath) =>
+          fse.existsSync(path.join(playgroundDir, "app", entryPath))
```

Comment:
You could simplify this to

```suggestion
        let serverEntry = ["tsx", "js", "jsx"].find(
          (entryPathExtension) =>
            fse.existsSync(path.join(playgroundDir, "app", `entry.server.${entryPathExtension}`))
```